### DISCUSSION TITLE
Read orientation

### DISCUSSION THREAD

**Discussion by Jediworm** [ablab/spades#919] [Q&A]
Hi.
I was hoping to ask a quick question. I am using Metaspades (I believe v3.15.3) on kbase. Do the input forward and reverse reads need to be ordered? I decontaminated my data outside kbase using Kneaddata and didn't use the "ordered" argument and so there are a few reads not in the same order. Thanks!

**Comment by asl** [comment]
Yes, your left reads should correspond to the right ones.
